server/src/test/java/com/linecorp/centraldogma/server/command/CreateSessionCommandTest.java

```diff
@@ -78,4 +80,18 @@ private static String toSerializedBase64(Object object) throws Exception {
             return Base64.getEncoder().encodeToString(baos.toByteArray());
         }
     }
+    
+    @Test
+    void testEquals() {
+        final Session session =
+                new Session("session-id-12345",
+                        "foo",
+                        Instant.EPOCH,
+                        Instant.EPOCH.plus(1, ChronoUnit.MINUTES),
+                        "abc");
+        CreateSessionCommand command = new CreateSessionCommand(1234L,
+                new Author("foo", "bar@baz.com"),
+                session);
+        assertFalse(command.equals("string"));
```

Comment:
```suggestion
        assertThat(command).isNotEqualTo("string")
```

Reply:
Can you also take a look at lint failures?

Reply:
I addressed the lint failures, thank you. I also updated to use `isNotEqualTo`, though I usually hesitate to do this. It's unclear whether `command.equals("string")` or `"string".equals(command)` (or both) will be called under the hood by testing framework. Here, we want to test the first so if testing framework actually invokes the second, then we would test `String.equals()` method instead. It's ok to use `isNotEqualTo` if you are sure that equality contract is correct, but here the contract is the thing we are testing. Test frameworks usually don't specify how exactly `isNotEqualTo`-like methods work, and behavior may change in future versions of testing frameworks.

I've posted a short twitter thread about this a while ago, you may [want to check](https://twitter.com/tagir_valeev/status/1265482450572136450). Normally, it's recommended to use [EqualsVerifier](https://github.com/jqno/equalsverifier) to check the equality contract, though this would be a much bigger pull request, and likely it will find more problems and I afraid I would not be able to fix all of them with zero experience in this project.

Reply:
My original comment was because if we use `assertj`'s built in method,  we get a richer error message on why the test failed (as opposed to just "`true` isn't equal to `false`")

Having said this, I agree with your point as well. Let me also check out [equalsverifier](https://github.com/jqno/equalsverifier) 🙇